Packages/com.unity.inputsystem/InputSystem/Devices/Pointer.cs

```diff
@@ -180,6 +183,12 @@ public class Pointer : InputDevice, IInputStateCallbackReceiver
         /// </remarks>
         public ButtonControl press { get; protected set; }
 
+        /// <summary>
+        /// The index of the display that was pointed.
```

Comment:
@jamesmcgill @EricFulton I wonder if we should call out here something like

"The index of the display associated with with this input event if supported by the platform, or the index of the primary display (zero) if not supported by the target platform. For more information see, <link to Unity or UGUI documentation or manual covering multi-display support for that platform>."

In order to avoid future bug reports on all platforms except Android until this is spread more widely.

Reply:
Excellent, marking this resolved.